lib/utils/isStandardSyntaxValue.js

```diff
@@ -6,18 +6,25 @@ const hasInterpolation = require("../utils/hasInterpolation")
  * Check whether a value is standard
  */
 module.exports = function (value/*: string*/)/*: boolean*/ {
+  let normalizedValue = value
+
+  // Ignore operators before variables (example -$variable)
+  if (value[0] === "-" || value[0] === "+" || value[0] === "*" || value[0] === "/") {
```

Comment:
You [don't even need to escape](http://jsbin.com/wanefagoyo/edit?js,console,output), if I'm not mistaken: `/^[-+*/]/`.